src/components/Invoices/index.js

```diff
@@ -0,0 +1,99 @@
+import InvoiceCards from "./InvoiceCards";
+import NewInvoice from "./NewInvoice";
+import ViewInvoice from "./ViewInvoice";
+import {useState, useEffect} from "react";
+
+
+const Invoices = () => {
+
+    const [invoices, setInvoices] = useState([]);
+
+    // --- fetch invoices and returns json promise ---
+    const extractResponseData = (response) => {
+        return response.json();
+    }
+
+    const fetchInvoices = async () => {
+        const response = await fetch('http://localhost:8080/invoices');
+        if (!response.ok) {
+            throw new Error();
+        }
+        return await extractResponseData(response);
+    }
+
+    // --- on mount logs the data retrieved ---
+    // --- Replace console.log(invoiceData) with setState inside the .then() to display data ---
+    useEffect( () => {
+        fetchInvoices()
+            .then((invoiceData) => {
+                setInvoices(invoiceData.data);
+            })
+            .catch((err) => {
+                err.message = 'Error! Could not resolve promise.';
+            });
+    }, []);
+
+    return (
+        <>
+            <div className="py-5 px-3">
+            <header className="d-flex justify-content-between align-items-end flex-wrap pt-3 px-0">
+                <div>
+                    <h1 className="fw-bolder">Invoices</h1>
+                    <p>There are 7 unpaid invoices</p>
```

Comment:
The number of invoices should be calculated.

Reply:
done